src/Presentation/Views/Registration/RegistrationViewModel.ts

```diff
@@ -1,4 +1,33 @@
+import { useState } from 'react';
+import { RegisterUserUseCase } from '../../../Domain/UseCase/User/RegisterUser';
+
 
 export default function RegistrationViewModel() {
-    
+    const [registerEmail, setRegisterEmail] = useState("");
+    const [registerPassword, setRegisterPassword] = useState("");
+
+    async function onClickRegister() {
+        try {
+            //Creates an account and logins automatically
+            const user = await RegisterUserUseCase(registerEmail, registerPassword);
+            console.log(user);
+        } catch (error: any) {
+            console.log(error.message);
+        }
+    };
+
+    function onChangeEmail(event: any) {
+        setRegisterEmail(event.target.value);
+    }
+
+
+    function onChangePassword(event: any) {
```

Comment:
nit, do you think that it would be possible to use a specific event as the type instead of any here?
https://felixgerschau.com/react-typescript-events/

I have no idea but it looks like maybe event: ChangeEvent<HTMLInputElement> could work?

Reply:
Yeah, we could look into that probably tomorrow. I know using the 'any' type is discouraged, but we just used that as a placeholder for now until we could find the appropriate type to put there.